website/docs/language/resources/syntax.mdx

```diff
@@ -29,22 +34,18 @@ resource "aws_instance" "web" {
 }
 ```
 
-A `resource` block declares a resource of a given type ("aws_instance")
-with a given local name ("web"). The name is used to refer to this resource
-from elsewhere in the same Terraform module, but has no significance outside
-that module's scope.
-
-The resource type and name together serve as an identifier for a given
-resource and so must be unique within a module.
-
 Within the block body (between `{` and `}`) are the configuration arguments
-for the resource itself. Most arguments in this section depend on the
-resource type, and indeed in this example both `ami` and `instance_type` are
-arguments defined specifically for [the `aws_instance` resource type](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance).
+for the resource itself. The arguments often depend on the
+resource type. In this example, both `ami` and `instance_type` are special 
+arguments for [the `aws_instance` resource type](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance).
 
 -> **Note:** Resource names must start with a letter or underscore, and may
 contain only letters, digits, underscores, and dashes.
 
+Resource declarations can include more advanced features, such as single 
+resource declarations that produce multiple similar remote objects, but only
+a small subset is required for initial use. You will learn more later in this page.
```

Comment:
```suggestion
a small subset is required for initial use. Refer to [Resource Arguments](#resource-arguments) for additional information.
```
Is this the topic you are alluding to? Our writing guidelines instruct us to treat each section on a page as a contained module of information. Our research indicates that people rarely read a page from beginning to end, so treating the content like a narrative can be confusing. It's better to link directly to a related section.

Reply:
Got you. Not sure where this refers to tbh. Used the original texts. If you are sure that this the place to link to, lets go for this anchor. @trujillo-adam

Reply:
Ah, I see. I think we should just remove the line, then. It was probably meant to refer to the article in its entirety, which is exactly counter the point I was making about writing it as a narrative. I'll modify my suggestion.